test/Interop/Cxx/function/Inputs/function-templates.h

```diff
@@ -0,0 +1,22 @@
+template <class T> T add(T a, T b) { return a + b; }
```

Comment:
So I already created a directory for class templates as `test/Interop/Cxx/templates`. Do you think it makes sense to put function template tests there too? I'm happy to rename existing tests to denote they are for class templates.

Reply:
Yeah, I was meaning to do this and forgot. I'll move these to `Cxx/templates/function-xxx`. I'll also create another PR that moves all the existing `Cxx/templates/` tests to `Cxx/templates/class-xxx`.

Reply:
Perfect, thank you!